src/libs/actions/Link.ts

```diff
@@ -157,4 +161,29 @@ function openLink(href: string, environmentURL: string, isAttachment = false) {
     openExternalLink(href);
 }
 
-export {buildOldDotURL, openOldDotLink, openExternalLink, openLink, getInternalNewExpensifyPath, getInternalExpensifyPath, openTravelDotLink, buildTravelDotURL};
+function buildURLWithAuthToken(url: string, shortLivedAuthToken?: string) {
+    const authTokenParam = shortLivedAuthToken ? `shortLivedAuthToken=${shortLivedAuthToken}` : '';
+    const emailParam = `email=${encodeURIComponent(currentUserEmail)}`;
+    const exitTo = `exitTo=${url}`;
```

Comment:
Hello ✋ 
Coming from https://github.com/Expensify/App/issues/55241
This has caused an issue on Desktop when an irl wasn't encoded properly, so invalid URL was opened in a browser
> Since the exitTo is a param and the link contains another param (nested param) and we don't encode it, the nested param isn't recognized as part of the exitTo link.

We resolved this with `const exitTo = `exitTo=${encodeURIComponent(url)}`;`